trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -28,11 +28,27 @@ const getCompany = async (req, res) => {
   }
 };
 
-const getPartyUrn = async ({ companyRegNo }) => {
+const getPartyUrn = async ({ companyRegNo, companyName }) => {
   if (!companyRegNo) {
     return '';
   }
 
+  if (process.env.AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED === 'true') {
```

Comment:
Are we able to use `isFeatureFlagEnabled` (in `libs/common/src/helpers/is-feature-flag-enabled.ts`) throughout?